File: centre-api/src/main/java/com/centre/poly/user/UserResponse.java
package com.centre.poly.user;

import com.centre.poly.role.Role;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;
import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class UserResponse {
    
    private Integer id;
    private String userName;
    private boolean isEnabled;
    private LocalDateTime createdDate;
    private List<Role> roles;
    private PersonResponse person;
}

File: centre-ui/src/app/services/models/user-response.ts
/* tslint:disable */
/* eslint-disable */
import { PersonResponse } from '../models/person-response';
import { Role } from '../models/role';
export interface UserResponse {
  createdDate?: string;
  enabled?: boolean;
  id?: number;
  person?: PersonResponse;
  roles?: Array<Role>;
  userName?: string;
}
